cmd/tempo-vulture/main.go

```diff
@@ -127,7 +127,7 @@ func main() {
 			}
 
 			// pick past interval and re-generate trace
-			seed := (generateRandomInt(startTime, currentTime) / interval) * interval
+			seed := (generateRandomInt(startTime, currentTime, rand.New(rand.NewSource(1))) / interval) * interval
 			rand.Seed(seed)
```

Comment:
Should this be changed to not use the global rand as well?

Reply:
I've updated the approach here.  The global rand is still used, but only to select from a list of known time slots, so I think it is acceptable.